sqlite/sqliteInt.h

```diff
@@ -3137,6 +3137,9 @@ struct AuthContext {
 #ifdef SQLITE_ENABLE_PREUPDATE_HOOK
 #define OPFLAG_ISNOOP        0x40    /* OP_Delete does pre-update-hook only */
 #endif
+/* vvv COMDB2 MODIFICATION vvv */
+#define OPFLAG_GENID         0x20    /* OP_Column only fetches genids */
+/* ^^^ COMDB2 MODIFICATION ^^^ */
```

Comment:
Nice comment 👍 . Can you put this kind of comments before/after other SQLite changes? It would make future SQLite upgrade easier.

Reply:
Thanks. Will do.